piety-promoted-v3/modernized/12-chapter-12.adoc

```diff
@@ -145,7 +145,10 @@ in the form of an address to the Lord, in acknowledgement of his mercies,
 which she wrote about the year 1788.
 A part of it is as follows:
 
-'`How has mine heart been led to meditate in your law,
+[.embedded-content-document.paper]
```

Comment:
Not sure whether to call this a paper or an address because of what it says above.

Reply:
I think paper is a smidge better. 👍